build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -180,8 +203,10 @@ class WatchImpl implements BuildState {
   /// Only one build will run at a time, and changes are batched.
   ///
   /// File watchers are scheduled synchronously.
-  Stream<BuildResult> _run(BuildEnvironment environment, BuildOptions options,
+  Stream<BuildResult> _run(BuildEnvironment env, BuildOptions options,
```

Comment:
I don't think we should shorten this

Reply:
Didn't mean to. Was supposed to be temporary for static analysis errors. Fixed.